htdocs/install/mysql/migration/17.0.0-18.0.0.sql

```diff
@@ -411,3 +411,4 @@ CREATE TABLE llx_c_invoice_subtype (
 
 ALTER TABLE llx_c_invoice_subtype ADD UNIQUE INDEX uk_c_invoice_subtype (entity, code);
 
+ALTER TABLE llx_fk_project ADD COLUMN parent int DEFAULT NULL;
```

Comment:
There is no table llx_fk_project. fk_project must be field name not table name

Reply:
Whoops! Thanks for catching that.